platform/MCU/rtl872x/inc/hw_ticks.h

```diff
@@ -23,8 +23,8 @@
 #include "platform_config.h"
 #include "system_tick_hal.h"
 
-#define SYSTEM_US_TICKS         (SystemCoreClock / 1000000) //cycles per microsecond
```

Comment:
My initial reaction is that `SystemCoreClock` is different on KM0 (ie prebootloader) vs KM4, but I dont think `SYSTEM_US_TICKS` is used in prebootloader, so I think this change is ok.

Reply:
Right, SYSTEM_US_TICKS is really only used by the System.ticks* APIs and one test case SPIX, which could/should probably be converted to use micros() later in a separate PR.